examples/README.md

```diff
@@ -539,10 +539,18 @@ ruby -run -ehttpd examples/wasm
 
 #### WebGL2 and WebGPU
 
-Bevy support for WebGPU is being worked on, but is currently experimental.
+Bevy ships with WebGPU by default, but it is being worked on and is currently experimental.
 
 To build for WebGPU, you'll need to disable default features and add all those you need, making sure to omit the `webgl2` feature.
```

Comment:
I'm wondering if this statement is still true... Wouldn't WebGPU be enabled by default if one does not define any features and roll with bevy defaults? I can verify this sometime this week but if someone knows... Please leave a comment!

Reply:
This is still true. The `webgl2` feature is included in our default features.

Reply:
Ah, thanks for the heads up. I re-reviewed the default feature list and now I understand. I'll adjust the language I've introduced in the previous paragraph accordingly.